testing/simulated/malicious_proposer_test.go

```diff
@@ -117,7 +153,7 @@ func (s *SimulatedSuite) TestProcessProposal_InvalidTimestamps_Errors() {
 	s.Require().Len(proposals, coreLoopIterations)
 	currentHeight := int64(blockHeight + coreLoopIterations)
 
-	// Prepare a valid block proposal, but 2 seconds in the future (i.e. attempt to roll timestamp forward)
+	// Prepare a block proposal, but 2 seconds in the future (i.e. attempt to roll timestamp forward)
```

Comment:
to give some context:
they point here is that one should not be able to push timestamp arbitrarily far in the future, otherwise a malicious proposer can stop the network by proposing timestamp = max.Uint64.
We try and bound the timestamp as strictly as possible (while maintaining its strictly monotonicity)